ceph-dashboard-nightly/config/definitions/ceph-dashboard-nightly.yml

```diff
@@ -0,0 +1,43 @@
+- job:
+    name: ceph-dashboard-nightly
+    project-type: freestyle
+    defaults: global
+    concurrent: true
+    node: huge && x86_64 && centos7 && rebootable
+    display-name: 'ceph: dashboard nightly'
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - build-discarder:
+          days-to-keep: 15
+          num-to-keep: 300
+          artifact-days-to-keep: -1
+          artifact-num-to-keep: -1
+      - github:
+          url: https://github.com/ceph/ceph/
+      - rebuild:
+          auto-rebuild: true
+
+    triggers:
+      - timed: '@midnight'
+
+    scm:
+      - git:
+          url: https://github.com/ceph/ceph.git
+          branches:
+            - origin/master
```

Comment:
I wouldn't do it like this because these two branches will run and it will be difficult to tell which one is failing. You want to have one of these jobs per branch. Instead of duplicating this job you can create a template and then define `branch` as a variable. The `ceph-volume` project does this for its nightly jobs here: https://github.com/ceph/ceph-build/blob/master/ceph-volume-nightly/config/definitions/ceph-volume-nightly.yml

Reply:
Ack. I'll refactor it.